events/equeue/equeue_platform.h

```diff
@@ -112,7 +112,7 @@ typedef struct equeue_sema {
     bool signal;
 } equeue_sema_t;
 #elif defined(EQUEUE_PLATFORM_MBED) && defined(MBED_CONF_RTOS_PRESENT)
-typedef unsigned equeue_sema_t[8];
+typedef unsigned equeue_sema_t[9];
```

Comment:
There was a reason... I can't remember it now. I'll try to replace it with our storage wrappers.